library/src/main/java/io/constructor/data/model/search/Group.kt

```diff
@@ -0,0 +1,6 @@
+package io.constructor.data.model.search
+
+import com.squareup.moshi.Json
+import io.constructor.data.model.ProductData
+
+data class Group(val value: String, val data: ProductData, @Json(name = "matched_terms") val matchedTerms: List<String>?)
```

Comment:
Not sure what this is used for.

Reply:
must have been development leftover, good call